dispatcher/dispatcher.py

```diff
@@ -35,7 +31,8 @@ def main():
     sh = daqnt.SignalHandler()
     SlackBot = daqnt.DaqntBot(os.environ['SLACK_KEY'])
     Hypervisor = daqnt.Hypervisor(control_mc[config['ControlDatabaseName']], logger,
-            daq_config['tpc'], vme_config, sh=sh, testing=args.test, slackbot=SlackBot)
+            daq_config, vme_config, control_inputs=config['ControlKeys'].split(), sh=sh,
+            testing=args.test, slackbot=SlackBot)
     MongoConnector = MongoConnect(config, daq_config, logger, control_mc, runs_mc, Hypervisor, args.test)
     DAQControl = DAQController(config, daq_config, MongoConnector, logger, Hypervisor)
```

Comment:
You probably also wanted to add 
`Hypervisor.mongo_connect = MongoConnector`
somewhere here?

Reply:
You mean like in the first line below this?

Reply:
🤦 , sometimes I oversee things in these tracked changes of git. Thanks